### DISCUSSION TITLE
Snapchat message recovery?

### DISCUSSION THREAD

**Discussion by Avrieknop86** [community/community#39330] [Other Feature Feedback, Questions, & Ideas]
Can you do it?

**Comment by jfullstackdev** [comment]
maybe you can ask questions regarding that here,

https://github.com/Snapchat

**Comment by jasonhill0123** [comment]
The PhotoBomb game concept you shared in image_0e6f60.png sounds like a hilarious way to elevate your group chats!
If you love exploring Snapchat's unique social dynamics, you should also check out [snapchat planets](https://snap-chat-planets.com/) to see how your closest friend rankings work.
